packages/main/src/MenuListItem.ts

```diff
@@ -11,30 +11,78 @@ import menuListItemCss from "./generated/themes/MenuListItem.css.js";
 /**
  * @class
  * @constructor
- * @extends StandardListItem
+ * @extends CustomListItem
  * @since 1.23.0
  * @private
  */
 @customElement({
 	tag: "ui5-menu-li",
-	template: StandardListItemTemplate,
-	styles: [StandardListItem.styles, menuListItemCss],
+	template: MenuListItemTemplate,
+	styles: [CustomListItem.styles, menuListItemCss],
 })
-class MenuListItem extends StandardListItem {
+class MenuListItem extends CustomListItem {
 	/**
 	 * Defines the associated MenuItem instance
 	 * @private
 	 */
 	@property({ type: Object })
 	associatedItem?: MenuItem;
```

Comment:
type: object has empty object for default value so we have to use ! instead of ?

Reply:
done